cookbooks/aws-parallelcluster-shared/libraries/test.rb

```diff
@@ -10,3 +10,7 @@ def redhat_on_docker?
 def rocky_on_docker?
   on_docker? && platform?('rocky')
 end
+
+def alinux2023_on_docker?
+  on_docker? && platform?('amazon') && node['platform_version'].to_i == 2023
```

Comment:
Can't `platform?('amazon') && node['platform_version'].to_i == 2023` be replaced by `os_properties.alinux2023?`?

Reply:
Since this is in test code, can we address this later?

Reply:
y